spec.bs

```diff
@@ -638,4 +741,17 @@ representing [=interest group/joining origins=], and whose [=map/values=] are [=
 
 </dl>
 
+<h3 dfn-type=dfn>Generate bid output</h3>
+
+The output of running a FLEDGE generateBid() script.
+
+<dl dfn-for="generate bid output">
+: <dfn>bid</dfn>
+:: A {{double}}. If the bid is zero or negative, then this interest group will not participate in
+  the auction.
+: <dfn>render</dfn>
+:: A [=URL=], which will be rendered to display the creative if this bid wins the auction.
+

```

Comment:
Probably want to add a couple things here:
The seller's origin, if this is the result of a component auction.
The interest group that generated this bid.  I think this will be important later.

Reply:
The current steps in the spec are not using these fields yet. Maybe we can add those in a future PR when they are needed? If some steps related to the fields are missing and we want to add in this PR, please let me know.

Reply:
In a future PR SGTM.